ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-For support and discussions, please use our [Discourse forums](https://discourse.mozilla.org/c/mozilla-voice-stt).
+For support and discussions, please use our [Discourse forums](https://discourse.mozilla.org/c/deep-speech).
```

Comment:
Do we need to ask for a new re-name of the section?

Reply:
Oh, good catch. We do, yeah. Who's responsible for this?